source/includes/_subscription.md

```diff
@@ -1402,7 +1402,26 @@ subscriptionApi.updateSubscriptionQuantity(subscriptionId, body, effectiveFromDa
 ```
 
 ```ruby
-TODO
+user = "demo"
+reason = nil
+comment = nil
+
+subscription                      = KillBillClient::Model::Subscription.new
+subscription.subscription_id      = "161692a4-c293-410c-a92f-939c5e3dcba7"
+subscription.bill_cycle_day_local = 16
```

Comment:
The `subscription.bill_cycle_day_local` field does not need to be set for the `update_quantity` method. I suggest removing it. Instead, the `subscription.quantity` should be set.